### DISCUSSION TITLE
Scrum Prep Milestone #3 - First Week

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#335] [Weekly Retrospective]
Weekly Development Update: First week of Milestone 3

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, November 1st

Thank you for your collaboration!

As for my collaboration:

Worked on M2 presentation, will resume meetings next week

**Comment by noel-vargas** [comment]
I didn't have time to work on the project this week

**Comment by NathBat09** [comment]
Not much, but thinking about what issues I can make for this milestone.

**Comment by misaelmercado1** [comment]
Brainstorming session for final lecture topic tasks and for last implementation details of the application

**Comment by RaquelQuinones** [comment]
Worked on the presentation and brainstormed new task for the milestone.

**Comment by williammartinez10** [comment]
Didn't work on the project this week.

**Comment by JoseLQV** [comment]
Brainstorming for new Ideas

**Comment by Chosen-Juan1** [comment]
Planning what to do for milestone 3.

**Comment by Rodal1** [comment]
Getting new ideas for next week.

**Comment by CaribbeanCool** [comment]
Didn't work much on the project

**Comment by JorgeC08** [comment]
Brainstorming for Lecture Topic Task (LTT)

**Comment by johnsantiago9** [comment]
Not much progress made, brainstorming new tasks

**Comment by JuanJRivera** [comment]
I didn't have time to work on the project this week.

**Comment by joel-alvarado** [comment]
Working out new tasks to create for next sprint.

**Comment by JorgeRivera94** [comment]
No progress this week. Will look to identify tasks for Milestone 3.

**Comment by ghost** [comment]
I had exams this week, so I couldnt get work done on the project.

**Comment by shenied-maldonado** [comment]
Slow week, but I focused on clearing up our tasks and setting up the foundation for the upcoming milestone.

**Comment by evnd04** [comment]
This week i couldnt work on the proyect due to exams, proyects and personaal issues. Will resume work tomorrow.

**Comment by Alejandro-DeJesus** [comment]
Didn't work much one project, devoted my attention towards other classes this week.

**Comment by yadrielsamir** [comment]
Worked on the tasks for the following weeks.

**Comment by Jjulianirizarry24** [comment]
Contributed to slide deck of milestone 2 presentation. Started planning for milestone 3. Will discuss with team next week.

**Comment by jonVargas1** [comment]
Didn't have much time to work on the project this week.

**Comment by lex939** [comment]
Late reply from my part, but I didn't get the chance to work on the project on this week. Will start next week.
